client-participation/.dockerignore

```diff
@@ -1,20 +1,8 @@
-node_modules
-bower_components
-public
-.DS_Store
+.sass-cache/*
 *.log
-js/tmpl/
```

Comment:
this is really weird... why was this ever in here!?

Reply:
not even sure how anything ever worked with this in here, since these files would be needed for compilation, right?

Reply:
Intermediate step in the build, only exists locally?

Reply:
Yes, it used to be a two step process managed through Gulp, these `.handlebars` templates would be compiled y Gulp into `.js` files in another directory - these are what the code actually imported and the original `.handlebars` files were not needed hence not copied across. Now it is all handled through Webpack Handlebars loader in a single step